src/plugins/vis_type_table/public/components/table_vis_controls.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { memo, useState, useCallback } from 'react';
+import { EuiButtonEmpty, EuiContextMenuItem, EuiContextMenuPanel, EuiPopover } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { DatatableRow } from 'src/plugins/expressions';
+import { CoreStart } from 'kibana/public';
+import { useKibana } from '../../../kibana_react/public';
+import { FormattedColumn } from '../types';
+import { Table } from '../table_vis_response_handler';
+import { exportAsCsv } from '../utils';
+
+interface TableVisControlsProps {
+  filename?: string;
+  cols: FormattedColumn[];
+  rows: DatatableRow[];
+  table: Table;
+  splitRow?: FormattedColumn;
+}
+
+export const TableVisControls = memo((props: TableVisControlsProps) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const togglePopover = useCallback(() => setIsPopoverOpen((state) => !state), []);
+  const closePopover = useCallback(() => setIsPopoverOpen(false), []);
+
+  const {
+    services: { uiSettings },
+  } = useKibana<CoreStart>();
+
+  const onClickExport = useCallback(
+    (formatted: boolean) =>
+      exportAsCsv(formatted, {
+        ...props,
+        uiSettings,
+      }),
+    [props, uiSettings]
+  );
+
+  const button = (
+    <EuiButtonEmpty
```

Comment:
Add an `aria-label` better explaining what will be exported and how. Ideally: `Export ${dataGridAriaLabel} as CSV` where the value of [`dataGridAriaLabel`](https://github.com/elastic/kibana/pull/70801/files?file-filters%5B%5D=.tsx#diff-53f83b72713e21fb0e0a0b9adc67e5babae912b71b5e0b4da5062b05f8a87b69R94) is consistent.

Reply:
Done!